bundles/ApplicationLoggerBundle/src/Controller/LogController.php

```diff
@@ -60,8 +60,8 @@ public function showAction(Request $request, Connection $db): JsonResponse
         $qb
             ->select('*')
             ->from(ApplicationLoggerDb::TABLE_NAME)
-            ->setFirstResult($request->get('start', 0))
-            ->setMaxResults($request->get('limit', 50));
+            ->setFirstResult($request->request->getInt('start', 0))
+            ->setMaxResults($request->request->getInt('limit', 50));
```

Comment:
I think you should also deprecate calling this via `GET`, because even if it's possible to send `GET` requests with a body, I think you shouldn't.

Reply:
You were also staying up late just for Pimcore uh? 🦉😄 💜 

Thank you, I've applied some change according to this input, i am almost sure i have missed other places(and bundles) too, will double-check this